sdk/storage/azure-storage-blob-changefeed/src/test/java/com/azure/storage/blob/changefeed/HelperSpec.groovy

```diff
@@ -0,0 +1,68 @@
+package com.azure.storage.blob.changefeed
+
+import com.azure.core.util.FluxUtil
+import com.azure.storage.blob.changefeed.models.BlobChangefeedEvent
+import com.azure.storage.blob.changefeed.models.BlobChangefeedEventData
+import com.azure.storage.blob.changefeed.models.BlobChangefeedEventType
+import com.azure.storage.blob.models.BlobType
+import reactor.core.publisher.Flux
+import spock.lang.Specification
+
+import java.nio.ByteBuffer
+import java.nio.channels.AsynchronousFileChannel
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.nio.file.StandardOpenOption
+import java.time.OffsetDateTime
+
+class HelperSpec extends Specification {
```

Comment:
Should this be `MockedChangeFeedSpec` or something more concrete? 

https://softwareengineering.stackexchange.com/questions/247267/what-is-a-helper-is-it-a-design-pattern-is-it-an-algorithm

Reply:
I think I may be able to get rid of this class entirely. I'll look into that as I re-review the tests